### PR TITLE
Create theme colours and styles

### LINKED ISSUE #59: Create theme colours and styles
Style the application to use Encore blue (Hex 064364) and the Encore logo throughout. 

For example, on the login page
- place a large copy of the Encore logo
encore-web-logo-colour (1)
(Encore logo alternative in white below -- please download to view)
Encore-lab-white-4x

Once logged in
- a small version of the Encore logo on the left side of the top navigation bar 
- colour the navigation bar Encore blue
(Small logo in white below -- please download to view)
encore-website-sm-logo (1)

For the canvas 
- white for the canvas and 
- light grey (e.g., #F8F8F8) for the background outside of the canvas

These are just some ideas -- feel free to explore your creative expression!

### PR DESCRIPTION
**Pull request by breity**
## Details
- Creates an Angular Material theme with complementary primary (#3A34B3) and accent (#6C63FF) colors. Applies primary color to toolbar and some UI elements, accent color to links and other controls and action buttons.
- Updates post color to #FFE663 (also complementary to the primary and accent colors).
- Adds some color and style to the main and project dashboards.
- Also adds icons to the project and board cards. (See what you think of these. We can always change.)
- Updates some board canvas styles and adds a grid background. (Again, see what you think of this. Grid doesn't move on pan or zoom as of now. May want to add in the future.)

Closes #59

### CODE REVIEW

File: src/app/services/post.service.ts [hunk 1]
```diff
@@ -11,68 +11,83 @@ interface Options {
...
-        let newLastItem = data.docs[data.docs.length - 1]
-        return { newLastItem, data }
-      })
+      .then((data) => {
+        let newLastItem = data.docs[data.docs.length - 1];
+        return { newLastItem, data };
+      });
+  }
+
+  getPostColor(): string {
```
Comment (Ammar-T) [inline src/app/services/post.service.ts:78]: `post.service.ts` is mainly for interacting with posts stored in Firebase. Could we move this colour into a `.scss` file and access it from there instead?

File: src/app/components/canvas/canvas.component.scss [hunk 2]
```diff
@@ -1,3 +1,12 @@
+// Grid colors
+$bg-color: #ffffff;
+$dot-color: #777777;
```
Comment (JoelWiebe) [inline src/app/components/canvas/canvas.component.scss:3]:
```suggestion
$dot-color: #d3d3d3;
```

File: src/app/components/canvas/canvas.component.scss [hunk 3]
```diff
@@ -1,3 +1,12 @@
...
+
+// Grid dot dimensions
+$dot-size: 1px;
```
Comment (JoelWiebe) [inline src/app/components/canvas/canvas.component.scss:6]:
```suggestion
$dot-size: 2px;
```

File: src/app/components/canvas/canvas.component.scss [hunk 4]
```diff
@@ -1,3 +1,12 @@
...
+$dot-space: 20px;
```
Comment (JoelWiebe) [inline src/app/components/canvas/canvas.component.scss:7]:
```suggestion
$dot-space: 40px;
```

**Comment by JoelWiebe** [comment]
Dashboard cards look great and project/board icons were a great idea. Looking sharp!

Having a grid-dot pattern is a neat idea. @Ammar-T is there a way to adjust those so they resize when we zoom in/out, like we do with background images?

Could we make the header style colour a bit more blue?

This is coming along great. Do you think we'll have time to add some mobile responsiveness before our go-live date of March 27th? Also, should we make our bucket and list-view modal pop-ups full screen with an x in the top-right corner?

File: src/app/services/post.service.ts (hunk 1 above)
Comment (Ammar-T) [reply src/app/services/post.service.ts:78]: Moving it to `utils/constants.ts` sounds good!

**Comment by JoelWiebe** [comment]
Updating the CSS dynamically with zoom and pan would be neat -- if we can figure that out!

Thanks @breity, so perhaps we can update the blue on this PR then treat the other items as new issues?

**Review by JoelWiebe — APPROVED** [review]
This looks great. Once that small change @Ammar-T suggested is made, let's merge what we have and worry about the blue colour and other features in a later PR.

**Comment by JoelWiebe** [comment]
@breity would you have time to close this one within the next few days so that we can enjoy this sweet UI! :)

**Pull request merged**